deploy/docker/start.sh

```diff
@@ -46,6 +43,12 @@ while [ $IDLE_TIME -lt 5 ]; do
     IDLE_TIME=$(expr $IDLE_TIME + 1)
     if curl http://localhost:${MGMT_PORT}/status >/dev/null 2>&1; then
         IDLE_TIME=0
+        # Print the latest erlang.log
+        erlang_logs=$(ps -ef |grep "tail -f /opt/emqx/log/erlang.log" |grep -v grep | sed -r "s/.*tail -f (.*)/\1/g")
+        new_erlang_log=$(echo $(ls -t /opt/emqx/log/erlang.log.*) | awk '{print $1}')
+        if [ -z "$(echo $erlang_logs | grep -o $new_erlang_log)" ];then
+            tail -f $new_erlang_log &
```

Comment:
This fork is then a leak.
Should record the pid of this fork and kill in the next round

Reply:
I did this because I was worried that some records would be lost between killing the old log and tailing the new log file, so I chose to tail out all the log files

Reply:
`tail -f somefile &` forks a new PID.
the forked linux process will keep running even if the parent process exited,
even after `somefile` has been deleted.
This is what I mean by 'leak'

Reply:
about the race: since our loop is every 5 seconds, it's very much unlikely that erlang.log will fill up and roll to a new file this soon.
it's by default 100KB. to be on the safe side, we can set it to 10M